api/controller.js

```diff
@@ -83,4 +78,32 @@ const addComment = (request,response,next) => {
 
 
 
-module.exports = {getTopics, getArticles, getArticlesById, getComments, addComment }
\ No newline at end of file
+const incrementVotes  = (request,response,next) => {
+    const id = request.params;
+    const voteIncrement = request.body;
+
+    updateVotes(id, voteIncrement)
+    .then(() =>{
+            return updateVotes(id, voteIncrement)
+        })
+        .then((articles) => {
+            response.status(202).send({articles});
+            
+        })
+        .catch(next);
+    
+   
+}
+
+const getUsers  = (request,response,next) => {
+    const {query} = request
```

Comment:
we don't need to grab any queries off the url for this one and once it's passed to the model it is not used so this can be deleted :)